package.json

```diff
@@ -44,11 +44,11 @@
     "@nestjs/schedule": "^2.2.0",
     "@nestjs/swagger": "^6.2.1",
     "@nestjs/typeorm": "^9.0.1",
+    "@polymeshassociation/local-signing-manager": "^3.0.1",
+    "@polymeshassociation/signing-manager-types": "^3.0.0",
     "@polymeshassociation/fireblocks-signing-manager": "^2.2.0",
     "@polymeshassociation/hashicorp-vault-signing-manager": "^2.1.0",
-    "@polymeshassociation/local-signing-manager": "^2.1.0",
-    "@polymeshassociation/polymesh-sdk": "23.0.0-alpha.27",
-    "@polymeshassociation/signing-manager-types": "^2.1.0",
+    "@polymeshassociation/polymesh-sdk": "23.0.0-alpha.34",
```

Comment:
I just merged v23 release. When that gets published this can get bumped to `"^23.0.0"`